backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/PatientUploadRow.java

```diff
@@ -71,33 +72,8 @@ public PatientUploadRow(Map<String, String> rawRow) {
 
   @Override
   public List<FeedbackMessage> validateRequiredFields() {
-    List<ValueOrError> allFields =
-        List.of(
-            firstName,
-            lastName,
-            middleName,
-            suffix,
-            race,
-            dateOfBirth,
-            biologicalSex,
-            ethnicity,
-            street,
-            street2,
-            city,
-            county,
-            state,
-            zipCode,
-            country,
-            phoneNumber,
-            phoneNumberType,
-            employedInHealthcare,
-            residentCongregateSetting,
-            role,
-            email);
-
-    List<FeedbackMessage> rowErrors = new ArrayList<>();
-    allFields.forEach(field -> rowErrors.addAll(field.getPossibleError()));
-    return rowErrors;
+    Field[] arrayOfField = PatientUploadRow.class.getDeclaredFields();
```

Comment:
could we use `this.getClass().getDeclaredFields()` and move this up to `FileRow` as well?